app/_src/gateway/get-started/services-and-routes.md

```diff
@@ -14,8 +14,35 @@ through the system.
 
 The high level overview below shows requests arriving at routes and being forward to services,
 with responses taking the opposite pathway:
+  
+<!--vale off-->
+
+{% mermaid %}
+flowchart LR
+  A(API client)
+  B("`Route 
+  (/mock)`")
+  C("`Service
+  (example_service)`")
+  D(Backend/
+  upstream API)
```

Comment:
```suggestion
  D(Upstream application)
```

Reply:
the doc uses 'upstream application', it feels like we should use that in the diagram too.